### ISSUE TITLE
relative barchart

### ISSUE THREAD

**Issue by mbannert** [mbannert/tstools#201]
@HomoCodens can you provide me with reproducible example of a relative bar chart. 
I have a hunch it ain't working. Is it just me?

**Comment by HomoCodens** [comment]
`tsplot(generate_random_ts(2, ranges_min = 0), left_as_bar = TRUE, relative_bar_chart = TRUE)`

The idea is not to scale all bars from 0 to 1 but normalize the whole graph.

Maybe reword the doc a bit to make that clear.

**Comment by mbannert** [comment]
agreed. This is solely a documentation issues. Maybe at some point we could a relative bar chart that does not relate to the global max but to the local max of each (stacked) bar. added clarification to the documentation and closed for now.
